runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -261,7 +261,7 @@ inlineVectorizedStringIndexOf(TR::Node* node, TR::CodeGenerator* cg, bool isUTF1
    const uint32_t elementSizeMask = isUTF16 ? 1 : 0;
    const int8_t vectorSize = cg->machine()->getVRFSize();
    const uintptrj_t headerSize = TR::Compiler->om.contiguousArrayHeaderSizeInBytes();
-   const bool supportsVSTRS = TR::Compiler->target.cpu.getSupportsVectorFacilityEnhancement2();
+   const bool supportsVSTRS = TR::Compiler->target.cpu.getSupportsVectorFacilityEnhancement2() && feGetEnv("TR_EnableVectorStringSearch") != NULL;
```

Comment:
Please open up an issue to get this addressed and add a TODO here (referencing the issue number) so it doesn't slip past our radar:

```
// TODO (#????): Re-enable VSTRS support in inlineVectorizedStringIndexOf
```